templates/AureliaSpa/ClientApp/app/components/fetchdata/fetchdata.ts

```diff
@@ -0,0 +1,27 @@
+/// <reference path="../../../../node_modules/aurelia-fetch-client/doc/whatwg-fetch.d.ts" />
+/// <reference path="../../../../node_modules/aurelia-fetch-client/doc/url.d.ts" />
+import {HttpClient} from 'aurelia-fetch-client';
+import {inject} from 'aurelia-framework';
+
+@inject(HttpClient)
+export class Fetchdata {
+    public forecasts: WeatherForecast[];
+    http: HttpClient;
```

Comment:
Why store this as a member property? It doesn't seem to be used anywhere outside the constructor.